docusaurus/docs/tutorials/ui-extensions/use-an-exposed-component.md

```diff
@@ -0,0 +1,31 @@
+---
+id: use-an-exposed-component
+title: Use an exposed component
+description: Reuse functionality from other plugins by using exposed components
+keywords:
+  - grafana
+  - plugins
+  - plugin
+  - extensions
+  - ui-extensions
+sidebar_position: 30
+---
+
+Exposed components are a way to share functionality between plugins.
```

Comment:
Since it appears to be confusing for people how this api differs from `addComponent`,  maybe we can add a sentence describing when this should be used? Something along those lines: 
`Use this feature when you want to share a react component with other app plugins, but you don't know or care where or under what circumstances it will be rendered.`